edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/identity/IUsernameParser.cs

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core.Identity
+{
+    public interface IUsernameParser
+    {
+        (string deviceId, string moduleId, string deviceClientType) Parse(string username);
```

Comment:
This is very MQTT Specific, right? Can we move it to the MQTT project? Or to Utils as MqttUsernameParser?

Reply:
Moved the parser to the MQTT projectd, and added reference to Hub.MQTT for Hub.MqttBrokerAdapter